src/Raven.Client/Documents/Indexes/IndexingPerformanceStats.cs

```diff
@@ -49,5 +49,6 @@ public IndexingPerformanceBasicStats(TimeSpan duration)
 
         public Size AllocatedBytes { get; set; }
         
+        public long DocumentsSize { get; set; }
```

Comment:
Can we use Size type here instead of long? We are doing this for other fields.

Reply:
done